network/incoming/incoming_network.go

```diff
@@ -3,14 +3,14 @@ package incomingNetwork
 import (
 	"bufio"
 	"fmt"
-	"github.com/GrappigPanda/Olivia/bloomfilter"
```

Comment:
This one's confusing and that's 100% my fault, but if you see "github.com/GrappigPanda/Olivia/bloomfilter", you can assume it's supposed to be there. The package name is "olilib" currently and I just created a ticket to replace it. Currently this is causing a build failure.